.github/workflows/ci-pipeline.yaml

```diff
@@ -8,7 +8,7 @@ on:
     branches: [ master ]
 
 env:
-  GO_VERSION: '1.16.1'
+  GO_VERSION: '1.18'
```

Comment:
I see that this previously included the minor version. Any thoughts on whether we should be continuing to do that (e.g. 1.18.9, or 1.19.4 if we're going for the latest stable one)? I'd guess it'd make the CI builds more consistent.

Reply:
convinced. stability over "main version tag". will update